go.mod

```diff
@@ -4,9 +4,14 @@ require (
 	github.com/fxamacker/cbor/v2 v2.2.0
 	github.com/go-kit/kit v0.8.0
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
+	github.com/go-playground/locales v0.13.0
+	github.com/go-playground/universal-translator v0.17.0
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/google/uuid v1.1.0
+	github.com/leodido/go-urn v1.1.0 // indirect
 	github.com/stretchr/testify v1.3.0
+	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
```

Comment:
There is apparently a v2 version of this library found [here](https://github.com/go-playground/assert). Given the `//indirect` comment another dependency probably pulled this in. Maybe the one below.

Reply:
[go-playground/assert v2.0.1](gopkg.in/go-playground/assert) is used in [go-playground/validator](https://github.com/go-playground/validator). This line has been removed after running `go mod tidy`.